backstop/scenarios/side-navigation.js

```diff
@@ -9,18 +9,18 @@ module.exports = [
     actions: [click('#test-expand')],
     selectors: ['#demo-side-navigation'],
   }),
+  scenario('Type submenu', {
+    actions: [click('#myDocumentsButton'), hover('#demo-take-hover')],
```

Comment:
What hovering `demo-take-hover` does? Because it is a simple `span`.

Reply:
`demo-take-hover` steals the hover effect after the click action takes place.  Otherwise the test image has a tooltip displaying over top of the submenu.

Reply:
In that case you can just hover/focus `body`, then no need to use additional IDs.

Reply:
Ahh yes, that is a much better idea.  Change made!